### ISSUE TITLE
INFO: Missing updates for heating groups

### ISSUE THREAD

**Issue by SukramJ** [SukramJ/hahomematic#554]
**custom_component/hahomematic version (if applicable):**
1.12.5 latest tested version / earlier versions are also affected

**Home Assistant version (if applicable):**
2022.7.7 latest tested version / earlier versions are also affected

**CCU version:**
- Model (CCU2/3, Raspi3/4, NUC, ProxmoxVE, HA-Addon ...): all CCU Derivated
- OS (Original, RaspberryMatic, debmatic ...): not relevant
- Version: 

**Describe the bug**
After a while (some days) the values of a group climate device are no longer updated in HA.

**Check your environment**
In CCU: Compare e.g. Temperature+"Last changed Timestamp" from your virtual device with your wall or radiator thermostat.
They should be (almost) synchronous. This is required to ensure that the problem is not related to the CCU.

**Possible Problem**
This problem could be caused by zombie-inits. Zombie-Init are not correctly de-initialized sessions that are still stuck in the CCU.
This could be caused by tools using the XMLRPC-API like e.g. Homematic-Manager, NodeRed, MobileApps, IoBroker(?) and Home-Assistant (core and custom integration).

Basically this integration tries to de-init the sessions on reload, restart and shutdown of HA, but this could fail due to a kill of HA.

**Check**
You can check if your problem is related to zombie-inits with the [script](https://homematic-forum.de/forum/viewtopic.php?f=31&t=62096&p=614148#) of @baxxy13.
If there are more sessions than expected from non-internal-ips under "Virt.-Devices Schnittstelle" then zombie-inits might be the reason.

**Solution**
In the case of zombie-inits a restart of the CCU is required.

**Comment by SukramJ** [comment]
One possible solution is to set `callback_port` to a fixed port(like `55677`). 
Hereby the integration reuses the previous `init` again.

**Comment by Bascht74** [comment]
Should this issue be closed (label "wontfix")?

**Comment by SukramJ** [comment]
No, this cannot be fixed, but people should read it, understand it and not add issues to this problem.
